scalability_and_performance/ztp-factory-install-clusters.adoc

```diff
@@ -0,0 +1,140 @@
+// Telcodocs-403 assembly
+:_content-type: ASSEMBLY
+[id="ztp-factory-workflows"]
+= Zero touch provisioning (ZTP) for factory workflows
+include::_attributes/common-attributes.adoc[]
+:context: ztp-factory-install
+
+:FeatureName: ZTP for factory workflows
+
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+toc::[]
+Zero touch provisioning (ZTP) for factory workflows accelerates the deployment of OpenShift with pre-certified hardware and software for rapid edge deployments.
+
+ZTP for factory workflows enables original equipment manufacturer (OEM) partners to pre-install OpenShift at their factory and build turnkey solutions on their hardware. This approach is well suited to a range of different industries such as healthcare, manufacturing, aerospace, to media, entertainment, retail, and telecommunications.
```

Comment:
The comma before and at the end is redundant.

Reply:
I checked with the style guide which runs a linter on this content and that comma called the oxford comma is needed. I can see why you think it is not and there is a debate about it but to adhere to our style guide we need it. https://en.wikipedia.org/wiki/Serial_comma